go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	emperror.dev/errors v0.7.0
 	github.com/IBM/dataset-lifecycle-framework/src/dataset-operator v0.0.0-20210201112050-4d77ac693ac1
 	github.com/IBM/go-sdk-core v1.1.0
```

Comment:
Unrelated, but wanted to ask a while ago - why do you import an old version and not v4 like satcon-go does? (I see they recently added v5 though).

Reply:
Good point. We actually don't need it. We should use the abstracted functionality from the library.